File: Java/UberVan.java
class UberVan extends Car{
    Map<String, Arraylist<String,Interger>> typeCarAccepted;
    Arraylist<String> seatsMaterial;

    public UberVan(String license, Account driver, Map<String, Arraylist<String,Interger>> typeCarAccepted, Arraylist<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;


    }
}

File: PHP/UberVan.php
<?php
require_once('car.php');
class UberX extends Car{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        parent::__construct($license,$driver, $typeCarAccepted, $seatsMaterial)
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
      } 

}
?>